src/main/java/appeng/api/implementations/tiles/IChestOrDrive.java

```diff
@@ -42,10 +42,4 @@ public interface IChestOrDrive extends ICellContainer, IGridHost, IOrientable {
      * @return if the device is online you should check this before providing any other information.
      */
     boolean isPowered();
-
-    /**
-     * @param slot slot index
-     * @return is the cell currently blinking to show activity.
-     */
-    boolean isCellBlinking(int slot);
```

Comment:
same here; mark it as deprecated but put in a default method for backwards compat